centreon/tests/e2e/features/Cloud-notifications/05-notification-get.feature

```diff
@@ -0,0 +1,28 @@
+@ignore
+@REQ_MON-24183
+Feature: List Notification Rules
+  As a Centreon user with access to the Notification Rules page
+  I want to view the list of notification rules
+  So that I can effectively organize my alerts in the monitoring system
+
+  Background:
+    Given a user with access to the Notification Rules page
+    And the user is on the Notification Rules page
+
+  Scenario: Empty notification rule list
```

Comment:
you may want to keep using "a user" or "the user" instead of "I" 
to be aligned with the other feature files.